vue/official/list/src/views/ViewMessagePage.vue

```diff
@@ -10,7 +10,7 @@
 
     <ion-content :fullscreen="true" v-if="message">
       <ion-item>
-        <ion-icon :icon="personCircle" color="primary"></ion-icon>
+        <ion-icon :aria-hidden="true" :icon="personCircle" color="primary"></ion-icon>
```

Comment:
```suggestion
        <ion-icon aria-hidden="true" :icon="personCircle" color="primary"></ion-icon>
```
`aria-hidden` is an attribute not a property, so it accepts the string `"true"` not the boolean.